res/values/strings.xml

```diff
@@ -396,11 +400,20 @@
     <string name="mailing_list">Mailing List</string>
     <string name="mailing_list_profile_info">Changes to mailing list name and image apply on this device only.</string>
 
+    <!-- broadcast lists -->
+    <string name="create_broadcast_list">Create Broadcast List</string>
+    <string name="menu_edit_broadcast_list">Edit Broadcast List</string>
+    <string name="menu_broadcast_list_name">Broadcast List Name</string>
+    <string name="broadcast_please_enter_broadcast_list_name">Please enter a name for the broadcast list.</string>
+    <string name="broadcast_list_warning">Broadcast Lists are always unencrypted because they use blind copy (BCC)</string>
+
     <!-- webxdc -->
     <!-- "Start..." button for an app -->
     <string name="start_app">Start…</string>
     <!-- this is a warning that is shown when one tries to send something to a chat that is not yet accepted. -->
     <string name="accept_request_first">Please accept the chat request first.</string>
+    <string name="show_webxdc_in_chat">Show Private App in Chat</string>
+    <string name="what_is_webxdc">What is Webxdc?</string>
```

Comment:
where is that shown? if in UI, ` as we are no longer using the term webxdc in UI, it should probably read `What are "Private Apps"?

Reply:
![](https://user-images.githubusercontent.com/18725968/201758046-c9f75ac0-11e4-4fbf-b776-caffa5a644fc.png)